### PR TITLE
Update front-end RFC and styles

### PR DESCRIPTION
**Pull request by jukrb0x**
leave some thoughts about implementing in Graphery with docker, plz review && correct.

add the frontend issues on indexing pages (Tutorials & Graphs) and practice code

add postgresql installation to the backend readme

### CODE REVIEW

**Review by FlickerSoul — CHANGES_REQUESTED** [review]
Please elaborate on your frontend RFC?

**Comment by jukrb0x** [comment]
The RFC is updated.

**Comment by jukrb0x** [comment]
This PR is way too chaotic, so far, we've done:
- add RFC for docker (low priority)
- update the RFC for frontend
-  update styles (to be continued)

File: graphery/src/views/Home.vue [hunk 1]
```diff
@@ -1,51 +1,71 @@
...
+    <!--          'flex-center': true,-->
+    <!--          'q-my-md': true,-->
+    <!--          row: $q.screen.gt.xs,-->
+    <!--          column: $q.screen.lt.sm,-->
+    <!--        }"-->
+    <!--      >-->
+    <!--        <div class="button-wrapper">-->
+    <!--          <q-btn size="lg" :to="{ name: 'Tutorials' }" exact>-->
+    <!--            {{ $t('nav.Tutorials') }}-->
+    <!--          </q-btn>-->
```
Comment (FlickerSoul) [inline graphery/src/views/Home.vue:50]: remove comments

**Review by FlickerSoul — APPROVED** [review]

**Pull request merged**